ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
     <filter root="/conf/hclecomm/settings/wcm" mode="update"/>
+    <filter root="/conf/hclecomm/sling:configs"/>
```

Comment:
@ShubhamGarg-AEM : why do we need this ?

Reply:
@rishabh-behl  /conf/hclecomm/sling:configs we have created a node at this path, this will not be created if we dont make this entry to filter.xml